Simplenote/CoreDataManager.swift

```diff
@@ -0,0 +1,102 @@
+import Foundation
+import CoreData
+
+@objcMembers
+class CoreDataManager: NSObject {
+
+    /// URL for the managed object model resource
+    ///
+    static private let modelURL: URL? = {
+        guard let path = Bundle.main.path(forResource: Constants.resourceName, ofType: Constants.resourceType) else {
+            return nil
+        }
+        return URL(string: path)
```

Comment:
```
Bundle.main.url(forResource: Constants.resourceName, withExtension: Constants.resourceType)!
```

If the Model cannot be loaded right away, good to crash. Absolutely nothing would work :(